cmd/edit/cluster/cmd.go

```diff
@@ -220,12 +287,201 @@ func run(cmd *cobra.Command, _ []string) {
 		disableWorkloadMonitoring = &disableWorkloadMonitoringValue
 	}
 
+	if len(cluster.AWS().SubnetIDs()) > 0 {
+		useExistingVPC = true
+	}
+	if useExistingVPC && !enableProxy && interactive.Enabled() {
+		enableProxyValue, err := interactive.GetBool(interactive.Input{
+			Question: "Update cluster-wide proxy",
+			Help: "To install cluster-wide proxy, you need to set one of the following attributes: 'http-proxy', " +
+				"'https-proxy', additional-trust-bundle",
+			Default: enableProxy,
+		})
+		if err != nil {
+			reporter.Errorf("Expected a valid proxy-enabled value: %s", err)
+			os.Exit(1)
+		}
+		enableProxy = enableProxyValue
+	}
+	if enableProxy {
+		err = interactive.PrintHelp(interactive.Help{
+			Message: "To remove any existing cluster-wide proxy value or an existing additional-trust-bundle value, " +
+				"enter a set of double quotes (\"\")",
+		})
+		if err != nil {
+			return
+		}
+	}
+
+	/*******  HTTPProxy *******/
+	if enableProxy && interactive.Enabled() {
+		var def string
+		if cluster.Proxy() != nil {
+			def = cluster.Proxy().HTTPProxy()
+		}
+		if httpProxy != nil {
+			def = *httpProxy
+			if def == "" {
+				// received double quotes from the user. need to remove the existing value
+				def = doubleQuotesToRemove
+			}
+		}
+		httpProxyValue, err = interactive.GetString(interactive.Input{
+			Question: "HTTP proxy",
+			Help:     cmd.Flags().Lookup("http-proxy").Usage,
+			Default:  def,
+		})
+		if err != nil {
+			reporter.Errorf("Expected a valid http proxy: %s", err)
+			os.Exit(1)
+		}
+
+		if len(httpProxyValue) == 0 { //user skipped the prompt by pressing 'enter'
+			httpProxy = nil
+		} else if len(httpProxyValue) == 2 { //user entered double quotes ("") to remove the existing value
```

Comment:
I think this should be
```suggestion
		} else if httpProxyValue == doubleQuotesToRemove { //user entered double quotes ("") to remove the existing value
```
no?